src/history/test/HistoryTests.cpp

```diff
@@ -811,7 +811,7 @@ TEST_CASE("catchup with a gap", "[history][catchupstall]")
  * Test a variety of orderings of CATCHUP_RECENT mode, to shake out boundary
  * cases.
  */
-TEST_CASE("Catchup recent", "[history][catchuprecent][!hide]")
+TEST_CASE("Catchup recent", "[history][catchup][catchuprecent]")
```

Comment:
This test is now failing, it looks like

Reply:
Works for me, could you send some logs?

Reply:
oh i think that catchup simulation times out, hence the intermittent failure. How's the performance of the test suite with this change?